R/utils.R

```diff
@@ -68,7 +68,16 @@ load_csv_files <- function(filenames, optionals, quiet = FALSE, ...) {
     header <- find_header(fqfn)
     col_types <- extract_header_info(header, label = "Column types:", fqfn, required = TRUE)
 
-    readr::read_csv(fqfn, comment = COMMENT_CHAR, col_types = col_types, ...) %>%
+    # Attempt the file read
+    op <- options(warn = 2)
```

Comment:
What does the `warn = 2` here?  Maybe a comment detailing the reason would be helpful

Reply:
👍